Hamming_Numbers/Hamming_Numbers.cpp

```diff
@@ -0,0 +1,48 @@
+//Program for finding first n hamming numbers for user input of n
+#include<iostream>
+#include<cmath>
+using namespace std;
+
+//Function for checking whether the number is hamming number or not
+bool is_hamming (int n) {
+    if (n == 1)
+    {
+        return true;
+    }
+    if (n % 2 == 0 || n % 3 == 0 || n % 5 == 0) {
+        for (int i = 2; i * i < n; ++i) {
+            if (n % i == 0) {
+                if (i % 2 != 0 and i % 3 != 0 and i % 5 != 0)
```

Comment:
Always use ``&&`` in place of "and", and ``||`` in place of "or".
Fix this in all files.